Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,21 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm.");
+            string TomsName = "Tom Jefferson";
+            int TomsAge = 19;
```

Comment:
Read input from console. Refer to Console.ReadLine, int.Parse, float,Parse, etc.